gradle/libs.versions.toml

```diff
@@ -34,6 +34,8 @@ room = "2.5.0-rc01"
 
 youtubedlAndroid = "a4afb4f6a2"
 
+commonsIoVer = "2.5"
```

Comment:
Which method needs this dependency?

Reply:
Actually no one, sorry. I added that lib for testing other methods of using SAF with inputStream and outputStream.